### PR TITLE
feat(sweeps): use launch to execute docker images

### PR DESCRIPTION
**Pull request by gtarpenning**
don't love this being in the sweep config but its the least bad option currently. 

https://wandb.atlassian.net/browse/WB-12474

### CODE REVIEW

File: src/sweeps/config/schema.json [hunk 1]
```diff
@@ -753,6 +753,10 @@
       "type": "string",
       "description": "Launch Job to run"
     },
+    "docker_image_uri": {
```
Comment (gtarpenning) [inline src/sweeps/config/schema.json:756]: might want to make this a subkey of a `launch_scheduler` section or something, and also include a key for `num_workers`

File: src/sweeps/config/schema.json [hunk 2]
```diff
@@ -753,6 +753,21 @@
       "type": "string",
       "description": "Launch Job to run"
     },
+    "launch": {
+      "type": "object",
+      "description": "Launch sweep-specific params",
+      "additionalProperties": false,
+      "properties": {
+        "image_uri": {
```
Comment (KyleGoyette) [inline src/sweeps/config/schema.json:761]: This should be top level like the program or the job

**Review by KyleGoyette — COMMENTED** [review]
Why not make the `image_uri` top level?

File: src/sweeps/config/schema.json [hunk 3]
```diff
@@ -753,6 +753,21 @@
       "type": "string",
       "description": "Launch Job to run"
     },
+    "image_uri": {
+      "type": "string",
+      "description": "Sweeps on Launch will use this uri instead of a job"
+    },
+    "scheduler": {
+      "type": "object",
+      "description": "Launch sweep-specific params",
```
Comment (KyleGoyette) [inline src/sweeps/config/schema.json:762]: Sweep scheduler specific parameters?

File: src/sweeps/config/schema.json [hunk 4]
```diff
@@ -753,6 +753,21 @@
       "type": "string",
       "description": "Launch Job to run"
     },
+    "image_uri": {
+      "type": "string",
+      "description": "Sweeps on Launch will use this uri instead of a job"
```
Comment (KyleGoyette) [inline src/sweeps/config/schema.json:758]: or program

**Review by KyleGoyette — APPROVED** [review]
2 nits

**Review by dannygoldstein — APPROVED** [review]

**Pull request merged**